### DISCUSSION TITLE
KiCad database libraries on Apple Silicon

### DISCUSSION THREAD

**Discussion by giscus** [cdwilson/cdwilson.dev#5] [Articles]
# KiCad database libraries on Apple Silicon

A new “Database Libraries” feature was recently added to the KiCad macOS Nightlies (6.99) and is scheduled to be officially released in KiCad 7.

In this article, I’m going to show you how to configure KiCad to use a SQLite-based database library with KiCad on macOS running on Apple Silicon.

https://cdwilson.dev/articles/kicad-database-libraries-on-apple-silicon/

**Comment by FastRabbit** [comment]
Works like a charm Thanks!

**Comment by SumantKhalate** [comment]
Thank for the wonderful tutorial!
I'm trying to get this working on Windows10 with Kicad V7, any idea what the connection string should be for windows?
P.S. I have the sqlite3 OBDC driver installed.

**Reply by cdwilson** [reply]
Unfortunately I don't have a Windows 10 setup I can try this on. You might find what you're looking for in this thread on the KiCad forums (if not, try asking there or the KiCad Discord):

https://forum.kicad.info/t/kicad-the-case-for-database-driven-design/34621

**Reply by SumantKhalate** [reply]
Yes, this forum post helped me resolve my problem.
I have documented the process for windows, in my [repo](https://github.com/SumantKhalate/KiCad-libdb)
